readme.md

```diff
@@ -91,7 +91,7 @@ var app = Elm.Todo.embed(node, flags);
 setupPorts(app.ports)
 ```
 
-So if you are interested in embedding Elm in something else, do the same trick! You can get more complete docs on embedding Elm in HTML [here](http://guide.elm-lang.org/interop/html.html) and JavaScript interop [here](http://guide.elm-lang.org/interop/javascript.html). Let the community know if you make something!
+So if you are interested in embedding Elm in something else, do the same trick! You can get more complete docs on embedding Elm in HTML [here](http://guide.elm-lang.org/interop/html.html) and JavaScript interop [here](https://guide.elm-lang.org/interop/). Let the community know if you make something!
 
```

Comment:
Confirmed old link is 404: http://guide.elm-lang.org/interop/javascript.html
![image](https://user-images.githubusercontent.com/194400/70790808-b5729100-1d8d-11ea-82de-0b7e9621cf30.png)

New link works: http://guide.elm-lang.org/interop
![image](https://user-images.githubusercontent.com/194400/70790836-c4594380-1d8d-11ea-8600-07c782cd03e0.png)

👍